bsc.typechecker/higherrank/DeepSeqCond.bs

```diff
@@ -20,6 +20,10 @@ f3 = Foo {x = 42; y = _when_ True 114}
 f4 :: Foo
 f4 = Foo {x = 42; y = _when_ False 114}
 
+-- Test calling primDeepSeqCond directly
```

Comment:
This wasn't included in `sysDeepSeqCond`, so no elaboration is being tested, just typecheck.  Is that what you intend?

And was there a particular experience or reasoning that made you want to include this?  Is this just to make sure that the function is in scope for users, or?

Reply:
I think @nanavati suggested that I test this, but I don't remember why.  Looks like I added this in commit cd03448a7dc57faec07db60c7d359ec122755739 separate from the rest of the test case.  

I can delete it or give it an actual check in the module.